core/src/test/scala/pickling/run/issue128.scala

```diff
@@ -0,0 +1,22 @@
+package scala.pickling.test.issue128
+
+import scala.pickling._, scala.pickling.Defaults._
+import scala.pickling.json._
+
+import org.scalatest.FunSuite
+
+case class A(intOpt: Option[Int]) {
```

Comment:
e.g. here we should try this with super dirty type parameters that are also "ok to pickle".